### ISSUE TITLE
The Light effect has been removed

### ISSUE THREAD

**Issue by 810** [niels9001/Fluent365#1]
On the latest Build you can't see the transparency. Should be updated to  XAML lighting

**Comment by niels9001** [comment]
Yeah, good point. Feel free to do a pull request if you already have a working fix.
